### PR TITLE
Disallow repacked package imports in checkstyle.xml

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

**Comment by leeavital** [comment]
:+1:

File: gradle-baseline-java-config/resources/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
         </module>
         <module name="HideUtilityClassConstructor"/> <!-- Java Coding Guidelines: Private constructors -->
         <module name="IllegalImport"> <!-- Java Coding Guidelines: Import the canonical package -->
-            <property name="illegalPkgs" value="sun, org.elasticsearch.common.base, org.elasticsearch.common.joda"/>
+            <property name="illegalPkgs" value="sun, org.elasticsearch.common.base, org.elasticsearch.common.joda, jersey.repackaged.com.google.common, com.google.api.client.repackaged"/>
```
Comment (leeavital) [inline gradle-baseline-java-config/resources/checkstyle/checkstyle.xml:102]: This should have a message along the lines of "don't use shadowed imports", no?

Comment (uschi2000) [inline gradle-baseline-java-config/resources/checkstyle/checkstyle.xml:102]: good idea, done.

Comment (bdworth) [inline gradle-baseline-java-config/resources/checkstyle/checkstyle.xml:102]: looks like I missed this party, but sorting the list entries might be a good idea

**Pull request merged**